haskell/test/Test/Ligo/BaseDAO/Common/StorageHelper.hs

```diff
@@ -75,13 +65,25 @@ getProposalEmulator addr pKey =
 -- | Note: Not needed at the moment, due to all the tests that uses this run only in emulator
 -- anyway. Commented due to weeder.
 
--- getTotalSupplyNetwork :: (Monad m) => Address -> FA2.TokenId -> NettestT m Natural
--- getTotalSupplyNetwork addr tokenId = do
---   fs <- getFullStorageView addr
---   let result = case M.lookup tokenId $ sTotalSupply (fsStorage fs) of
---         Just v -> v
---         Nothing -> error "getTotalSupply: token_id does not exist."
---   pure result
+------------------------------------------------------------------------
+-- CheckBalanceFn
+------------------------------------------------------------------------
+
+type CheckBalanceFn m = Address -> Address -> Natural -> m ()
+
+checkBalanceEmulator :: Address -> Address -> Natural -> EmulatedT PureM ()
+checkBalanceEmulator addr owner bal = do
+  fh <- getFreezeHistoryEmulator addr owner
+  (sumAddressFreezeHistory <$> fh) @== Just bal
+
+
+checkBalanceNetwork :: (Monad m) => Address -> Address -> Natural -> NettestT m ()
+checkBalanceNetwork _addr _owner _bal = do
+  -- We can't check BigMap in network
+  True @== True
```

Comment:
I get that we cannot check balance in network, but how does this help?

Reply:
This allows using the same test case, and run it in both network and emulator:

https://github.com/tqtezos/baseDAO/pull/275/files#diff-bce87f85bb8daa411eca6841102ce815e6bccc0b9977a22c827f936f12559810R124-R129

I could also just use `(\ _ _ _ -> pure ())`, but it is mainly for consistency purpose.

Reply:
@rinn7e But AFAIU it does not check anything, so wouldn't it make the test useless?

Reply:
Using this test as example: https://github.com/tqtezos/baseDAO/pull/275/files#diff-8ce82adc395e63130a569731f81e84d545f091f9038d4bb597c84aed02e0c221R28-R38

Basically, this test does a bunch of things including checkBalance. When running in network, it just skip the checkBalance, but it still run the rest of the actions.

Reply:
Ok, may be we should rename the function to something like "checkBalanceNetworkDummy", or use 

>  (\ _ _ _ -> pure ())

inline.

Reply:
I've updated this now.